roundware/api2/serializers.py

```diff
@@ -342,10 +342,16 @@ def to_representation(self, obj):
 
 
 class SpeakerSerializer(serializers.ModelSerializer):
+    children = serializers.SerializerMethodField()
+
     class Meta:
         model = Speaker
         fields = "__all__"
 
+    def get_children(self, obj):
+        # Access the reverse relationship via `children`
+        return [child.id for child in obj.children.all()]  # Return IDs of children
```

Comment:
i think you can do `return obj.children.values_list('id')` instead

Reply:
One small, but critical change to get the proper output format:
`return obj.children.values_list('id', flat=True)`